app/src/main/java/fr/free/nrw/commons/filepicker/FilePicker.java

```diff
@@ -201,7 +201,7 @@ private static boolean isPhoto(Intent data) {
     }
 
     private static Intent plainGalleryPickerIntent() {
-        Intent intent = new Intent(Intent.ACTION_GET_CONTENT);
+        Intent intent = new Intent(Intent.ACTION_OPEN_DOCUMENT);
```

Comment:
Could you please add a comment explaining in details how we came to this? Do not hesitate to link to existing GitHub comments, the Android bug tracker, etc. Don't hesitate to write a long comment with several paragraphs if you need. Thanks!

Reply:
I have explained the change in a comment.